src/components/cylc/common/sort.js

```diff
@@ -68,21 +63,29 @@ export const DEFAULT_COMPARATOR = (left, right) => {
  * name, but that respects natural order for numbers, i.e. [1, 2, 10].
  * Not [1, 10, 2].
  *
- * @param {Object[]} array - list of string values, or of objects with string values
- * @param {Object} value - a value to be inserted in the list, or an object wrapping the value (see iteratee)
- * @param {SortedIndexByIteratee=} iteratee - an optional function used to return the value of the element of the list}
- * @param {SortedIndexByComparator=} comparator - function used to compare the newValue with otherValues in the list
+ * @param {Object[]} array - list of string values, or of objects with string
+ * values
+ * @param {Object} value - a value to be inserted in the list, or an object
+ * - wrapping the value (see iteratee)
+ * @param {SortedIndexByIteratee=} iteratee - an optional function used to
+ * - return the value of the element of the list}
+ * @param {SortedIndexByComparator=} comparator - function used to compare
+ * - the newValue with otherValues in the list
  * @return {number} - sorted index
  */
-export function sortedIndexBy (array, value, iteratee, options = {}) {
+export function sortedIndexBy(array, value, iteratee, options = {}) {
   // comparator, reverse = false) {
   if (array.length === 0) {
     return 0
   }
   // If given a function, use it. Otherwise, simply use identity function.
   const iterateeFunction = iteratee || ((value) => value)
-  // If given a function, use it. Otherwise, simply use locale sort with numeric enabled
-  const comparatorFunction = options.comparator || ((leftObject, leftValue, rightObject, rightValue) => DEFAULT_COMPARATOR(leftValue, rightValue))
+  // If given a function, use it. Otherwise, simply use locale sort with
+  // numeric enabled
+  const comparatorFunction
+    = options.comparator
+    || ((leftObject, leftValue, rightObject, rightValue) =>
+      DEFAULT_COMPARATOR(leftValue, rightValue))
```

Comment:
Can't say I'm a fan of line break before assignment operators 🤮 But I've had a look and can't see any way to set `operator-linebreak` to `after` for assignment operators only.

(P.S. apparently I am in the minority but I generally prefer line breaks after logical operators too, reason being my brain reads a line as a "unit", so if there is no operator at the end my brain assumes that's the end of the statement, only to discover there is another condition when my eyes jump to the next line. Whereas if there is an operator at the end I know in what context I need to process the next line (including whether it could be skipped to due short-circuiting))